modules/persistent-storage-csi-cloning-overview.adoc

```diff
@@ -3,19 +3,27 @@
 // * storage/container_storage_interface/persistent-storage-csi-cloning.adoc
 
 [id="persistent-storage-csi-cloning-overview_{context}"]
-= Overview of CSI volume cloning
+= How CSI volume cloning works
 
-A Container Storage Interface (CSI) volume clone represents a duplicate of an existing persistent volume that is its own unique volume on the system at a particular point in time. With cloning, the back-end device creates an exact duplicate of the specified volume, rather than creating a new empty volume. After it has been dynamically provisioned, a volume clone can be consumed as any standard volume would be.
+Cloning creates an exact duplicate of the specified volume on the back-end device, rather than creating a new empty volume. Once dynamically provisioned, you can use a volume clone just as you would use any standard volume.
```

Comment:
Drive-by comment, here, but "once" should only be used to mean "one time" per ISG.

Reply:
Ah, good catch! Fixed, thanks!